README.md

```diff
@@ -75,3 +72,7 @@ But the following `invalid.md` is **not** ok:
 ```md
 Here is a [bad relative link](wooorm/remark-dead-link)
 ```
+
+## License
+
+MIT © [David Clark](https://github.com/davidtheclark)
```

Comment:
I don't think there's a need to add this section. Why do you?

Reply:
Most npm package readmes include a `License` section these days, and it just seemed like an omission. I don't feel strongly one way or another; I was just taking a pass through the readme file to update it and noticed it was missing a license.

Happy to remove this section if you'd like.

Reply:
Yeah let's just remove it.